freqtrade/wallets.py

```diff
@@ -0,0 +1,44 @@
+# pragma pylint: disable=W0603
+""" Wallet """
+import logging
+from typing import Dict, Any, NamedTuple, Optional
+from collections import namedtuple
+from freqtrade.exchange import Exchange
+
+logger = logging.getLogger(__name__)
+
+
+class Wallet(NamedTuple):
+    exchange: Optional[str] = None
+    currency: Optional[str] = None
```

Comment:
instead of `Optional` - don't assign a default value, it cannot be used anyway as there is only one way to genreate a `Wallet` - which is by adding both exchange AND Currency (see 20 lines below)

Rest looks good :+1: